### ISSUE TITLE
Title page and titles without space between #and text are not displayed correctly.

### ISSUE THREAD

**Issue by carllapa** [claudioc/jingo#93]
The Markup code:

![jingo title problem_02](https://cloud.githubusercontent.com/assets/5005186/9177545/123e6a14-3f92-11e5-89c8-77df44a4236d.png)

The result:

![jingo title problem_01](https://cloud.githubusercontent.com/assets/5005186/9177544/123d9cba-3f92-11e5-9f59-fba8767f8761.png)

As you can see in the images, when a title doesn't have a blank space after # it's not displayed ok.

I've installed latest version of Jingo using npm v2.13.4 and node v0.12.2.

**Comment by claudioc** [comment]
Hi,

I will take a closer look at this issue, but I am afraid it's a problem with the Markdown parser (which I didn't write).

**Comment by J3rome** [comment]
The problem is located at line 92 of the file /routes/wiki.js
```javascript
res.render("show", {
        page: page,
        title: app.locals.config.get("application").title + " – " + page.title,
        content: renderer.render("#" + page.title + "\n" + page.content)
      });
```
There is simply a space missing between the # and the page.title

Changing the line content to
```
content: renderer.render("# " + page.title + "\n" + page.content)
```
Fix the problem

**Comment by achimwessling** [comment]
Yeah, that's right. A had this problem also and solved it the same way.

**Comment by carllapa** [comment]
Thank you both for the answer, you saved me a lot of time of debbuging the code :)

**Comment by claudioc** [comment]
Uhm, interesting. I cannot replicate the problem on the current version. Do you mind trying with the current version?

**Comment by J3rome** [comment]
I am using the latest version. 
I forked the [Repo](https://github.com/J3rome/jingo/) couple days ago

**Comment by claudioc** [comment]
Damn... node 0.12 on OSX 10.10.4

<img width="201" alt="screen shot 2015-09-02 at 20 01 17" src="https://cloud.githubusercontent.com/assets/166692/9639671/6e9935d2-51ad-11e5-95c0-c63cba80a89c.png">

<img width="196" alt="screen shot 2015-09-02 at 20 02 02" src="https://cloud.githubusercontent.com/assets/166692/9639687/80e5e2a8-51ad-11e5-98be-c8865a1f51f6.png">

**Comment by J3rome** [comment]
Running it on node 0.10.25 on ubuntu 14.04.
I will try to upgrade it soon and see if I get the same behaviour has you

**Comment by achimwessling** [comment]
I'm on window$ with iojs 3! So this is probably the difference.

I also replaced iconv with iconv-lite, because the installation process of
iconv on window$ isn't a fun. So if one of you is interested he is welcome
to look at
https://github.com/achimwessling/jingo/commit/06a09a796e12968b3158f8c9f691b1b220f10d77
.

Achim

J3rome <notifications@github.com> schrieb am Mi., 2. Sep. 2015 um 20:25 Uhr:

**Comment by claudioc** [comment]
I upgraded to Node 4.1.x and now I have this problem. Going to fix it.
@achimwessling interesting idea, using iconv-lite! Will try to add it to 1.3.2
Thanks to you all

**Comment by claudioc** [comment]
Fixed in 1.4.0 and removed iconv completely (with https://github.com/andyhu/node-transliteration), because this is what I used iconv for (iconv-lite does not transliterate)

**Comment by achimwessling** [comment]
Thanks Claudio,

looks great!

Achim

Claudio Cicali <notifications@github.com> schrieb am So., 11. Okt. 2015
21:03:
